src/models/Settings.js

```diff
@@ -144,6 +144,16 @@ export default Model.extend({
   },
 
   derived: {
+    showPasswordToggle: {
+      deps: ['features.showPasswordToggleOnSignInPage'],
+      fn: function () {
+        // showPasswordToggle is for OIE only.
+        // Used to default showPasswordToggleOnSignInPage to true.
+        return _.isUndefined(this.options.features?.showPasswordToggleOnSignInPage)
```

Comment:
`features` may also be undefined.

Reply:
would this work `this.get('features.showPasswordToggleOnSignInPage') !== false` ?

Reply:
> would this work `this.get('features.showPasswordToggleOnSignInPage') !== false` ?

This is not the correct logic because if we set it to true it will return false, and we would want to return true.
Cases:
- features.showPasswordToggleOnSignInPage: true, showPasswordToggle: true
- features.showPasswordToggleOnSignInPage: false, showPasswordToggle: false
- features.showPasswordToggleOnSignInPage: undefined, showPasswordToggle: true